core/extraction/towns_fund_round_two.py

```diff
@@ -636,6 +641,40 @@ def extract_funding_data(df_input: pd.DataFrame) -> pd.DataFrame:
         "End_Date",
     ]
     df_funding_data.drop_duplicates(subset=no_duplicates, keep="first", inplace=True)
+
+    # drop always unused cells for funding secured before 2020 and beyond 2026
+    unused_mask = df_funding_data.loc[
```

Comment:
Are these 2 bits of code (R1 / R2) similar enough to move into a common function? If so, it could go into extraction/utils.py

Reply:
This is what I have done thanks